plugins/animate/README.md

```diff
@@ -0,0 +1,113 @@
+# animate
```

Comment:
Are there plans to move to a docs folder and just the docs for this? Seems like it would work really well.

Reply:
hah https://github.com/vigetlabs/tailwindcss-plugins/issues/16
absolutely agree

Reply:
👏